requirements-full.txt

```diff
@@ -3,16 +3,14 @@ AccessControl==6.3
 Acquisition==5.2
 AuthEncoding==5.0
 BTrees==5.2
-Chameleon==4.2.0; python_version == '3.7'
 Chameleon==4.4.4; python_version == '3.8'
 Chameleon==4.4.4; python_version > '3.8'
```

Comment:
Same version for `3.8` and `> 3.8`?

Reply:
Yes, please see https://github.com/zopefoundation/Zope/pull/1214#discussion_r1619117320